test/services/AuditServiceSpec.scala

```diff
@@ -42,8 +42,12 @@ class AuditServiceSpec extends AnyWordSpec with Matchers with MockAuditConnector
   "auditIndividualJourney" should {
     "send an event" when {
       "the entity is an individual and identifiers match" in {
+        mockRetrieveFullName(testJourneyId)(Future.successful(Some(testFullName)))
```

Comment:
I don't think we need to mock "retrieveFullName", retrieveDateOfBirth" or "retrieveNino" because they are not invoked by